apps/ledger-live-mobile/e2e/page/accounts/assetAccounts.page.ts

```diff
@@ -0,0 +1,49 @@
+import { expect } from "detox";
+import {
+  currencyParam,
+  openDeeplink,
+  waitForElementById,
+  getElementById,
+  scrollToId,
+  tapById,
+} from "../../helpers";
+
+const baseLink = "account";
+
+export default class AssetAccountsPage {
+  assetBalance = () => getElementById("asset-graph-balance");
+  titleId = (assetName: string) => `accounts-title-${assetName}`;
+  accountAssetId = (assetName: string) => `account-assets-${assetName}`;
+  accountRowId = (accountId: string) => `account-row-${accountId}`;
+  accountNameRegExp = /account-row-name-.*/;
+
+  @Step("Wait for asset page to load")
+  async waitForAccountPageToLoad(assetName: string) {
+    await waitForElementById(this.titleId(assetName.toLowerCase()));
+  }
+
+  @Step("Expect asset balance to be visible")
+  async expectAccountsBalanceVisible() {
+    await expect(this.assetBalance()).toBeVisible();
+  }
+
+  async expectAccountsBalance(expectedBalance: string) {
```

Comment:
Should we add some @step here ? (and for the next one)

Reply:
I don't use them in Speculos tests, they are from mocked tests, that's why I didn't add the decorator.